### ISSUE TITLE
Sorry, IPhone 8 (GSM)is not supported on iOS at this point

### ISSUE THREAD

**Issue by shmistan11** [checkra1n/BugTracker#2168]
**Tell us about your setup:**
1. What iDevice are you using?
iPhone 8
2. On what version of iOS is it?
14.7.1
3. What version of checkra1n are you using?
Checkn1x 1.1.7
4. What is your host system (OS version? Hackintosh? VM? etc.)?
Windows
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)?
USB-A to lightning
**What are the steps to reproduce the issue?**
1. Boot into Checkn1x 
2. Connect iPhone 8 and go into options 
3.check untested iOS versions and bypass a11 check
...

**What do you expect, and what is happening instead?**
I expect it to let me proceed and jailbreak my device but the start button is grayed out

**Did you see a popup on the device stating it entered "Safe Mode" due to an error?**
No

**Does the issue also occur if you tick "Safe Mode" in the checkra1n options?**
Yes

**Any other info, error logs, screenshots, ...?**yes
https://www.filemail.com/d/glywupcawvmxwod

**Comment by teelekkung** [comment]
have you enabled allow untested ios in options menu??

**Comment by shmistan11** [comment]
Yes

**Comment by teelekkung** [comment]
how about cli mode

**Comment by shmistan11** [comment]
I don’t know how to use it

**Comment by worf1337** [comment]
make sure Skip A11 BPR Check is enabled.

**Comment by shmistan11** [comment]
It is, I’ve tried a lot of times

**Comment by shmistan11** [comment]
I've tried every OS, too, all have errors like that or completely different ones, but I am not going to give up
.

**Comment by vadimszzz** [comment]
Download latest checkra1n 0.12.4
https://checkra.in/releases/0.12.4-beta

Check [+] Allow untested devices and [+] Skip A11 BPR Check

@Siguza close the issue thx

**Comment by mbiracalvin** [comment]
> I've tried every OS, too, all have errors like that or completely different ones, but I am not going to give up
> .

Did you manage???

**Comment by shmistan11** [comment]
No, it would always leave the error message or say lockdownd error. Tried on a mac but it got stuck jailbreaking.

**Comment by xubenchu0610** [comment]
me，too
